src/components/pages/SurveyPage.jsx

```diff
@@ -136,15 +95,85 @@ function SurveyPage() {
           value={formData.maritalStatus}
           name="maritalStatus"
           options={MARITAL_STATUSES}
-          onChange={handleInputChange}
         />
       </div>
 
       <div className="actions">
-        <button className="left" onClick={handleClearForm}>Clear Form</button>
-        <button className="right" onClick={handleSubmit}>Submit</button>
+        <button type="button" className="left" onClick={handleClearForm}>Clear Form</button>
+        <button type="submit" className="right">Submit</button>
       </div>
-    </main>
+    </Form>
+  )
+}
+
+/**
+ * Defines a function to map Formik props to form values
+ * Function name matches Formik option key mapPropsToValues
+ * @param {} props - includes email and password
+ * @returns {object} - formatted field values
+ */
+export function mapPropsToValues ({ race }) {
+  return {
+    race: race || [],
+  }
```

Comment:
We will need to add all the other inputs to this mapping of props to field names with a default.